charts/harvester-csi-driver/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   harvester:
     csiDriver:
       repository: rancher/harvester-csi-driver
-      tag: ""
+      tag: "v0.1.1"
   csi:
```

Comment:
As noted in https://github.com/rancher/charts/pull/1595#discussion_r750489873, we need to use mirrored images for longhorn.